README.md

```diff
@@ -37,6 +37,10 @@ via `--motd` key. It's possible to use the following variables surrounded by
 `{{ }}`:
 - `start_date`: time when the node was started
 - `uptime`: uptime in the format 'XX days XX hours XX minutes'
+- `tcp_in`: counter of tcp incoming packets
```

Comment:
Maybe rename it to `tcp_packets_in`, etc? Because this also can mean bytes count, which can be added later.